src/commands/update/datacenter.ts

```diff
@@ -29,15 +31,12 @@ async function update_datacenter_action(options: UpdateDatacenterOptions, name:
     const allEnvironments = await command_helper.environmentStore.find();
     const datacenterEnvironments = allEnvironments.filter((e) => e.datacenter === name);
 
-    const targetGraph = await newDatacenter.enrichGraph(new CloudGraph());
+    const graph = new CloudGraph();
 
-    for (const record of datacenterEnvironments) {
-      const originalEnvGraph = await record.config?.getGraph(record.name, command_helper.componentStore);
-      const targetEnvGraph = await newDatacenter.enrichGraph(originalEnvGraph || new CloudGraph(), record.name);
+    const vars = await command_helper.promptForVariables(graph, newDatacenter.getVariables());
+    newDatacenter.setVariableValues(vars);
```

Comment:
Does this fix https://github.com/architect-team/arcctl/issues/62 by chance?

Reply:
Yes it does